src/site/_includes/js/eligibility.js

```diff
@@ -2693,6 +2708,28 @@ function clipperStartResult(input) {
   return program.getResult();
 }
 
+function homelessPreventionSystemResult(input) {
+  const grossLimit = MonthlyIncomeLimits.fromAnnual(
+    cnst.homelessPreventionSystem.ANNUAL_INCOME_LIMITS,
+    cnst.homelessPreventionSystem.ANNUAL_INCOME_LIMIT_ADDL_PERSON);
+
+  const riskLosingHousing = isOneOf(input.housingSituation, [
+    'unhoused-risk']);
+
+  const incomeLimit = grossLimit.getLimit(input.householdSize);
+  const underIncomeLimit = le(grossIncome(input), incomeLimit);
+  const program = new Program();
+
+  program.addCondition(new EligCondition(
+    `Have a gross income below ${usdLimit(incomeLimit)} 
+    per month`,
+    underIncomeLimit));
+  program.addCondition(new EligCondition(
+    'Be at risk of losing your housing in the next 14 days',
```

Comment:
I suggest also removing the 14 days bit from here (see comment in eligibility.liquid)